packages/terra-functional-testing/src/services/wdio-visual-regression-service/VisualRegressionLauncher.js

```diff
@@ -62,8 +63,34 @@ class VisualRegressionLauncher {
   /**
    * Function to be executed after a test in Mocha.
    * @param {Object} test - test details
+   * @param {String} test.parent - test parent title
+   * @param {String} test.title - test title
+   * @param {Object} results - results object
+   * @param {boolean} results.passed - indicates if the test passed.
    */
-  afterTest() {
+  afterTest({ parent, title }, _, { passed }) {
+    if (!passed) {
+      const { config } = browser;
+      const { formFactor } = config.launcherOptions;
+      const screenshotContext = {
+        desiredCapabilities: this.context.desiredCapabilities,
+        suite: this.currentSuite,
+        test: this.currentTest,
+        meta: {
+          currentFormFactor: formFactor,
+        },
+        options: {
+          name: `${parent}_${title}`,
```

Comment:
Does the visual regression service sanitize this string? There's a handful of characters/spaces that will throw errors when used in a filename

Reply:
Yes it does. This is utilizing the same utils used for creating the reference and screenshot file paths.